src/e2e-test/resources/pluginParameters.properties

```diff
@@ -114,3 +115,13 @@ account.schema= [{"key":"Id","value":"string"},{"key":"IsDeleted","value":"boole
   {"key":"JigsawCompanyId","value":"string"},{"key":"AccountSource","value":"string"},\
   {"key":"SicDesc","value":"string"}]
 
+CreateBQTableQueryFile=BigQuery/BigQueryCreateTableQuery.txt
+InsertBQDataQueryFile=BigQuery/BigQueryInsertDataQuery.txt
+
+bigQueryDatatypesColumnsList=(Id,Name,Col_Timestamp__c,Col_Date__c,Col_Currency__c,Col_Email__c,Col_Number__c,\
+  Col_GeoLocation__Latitude__s,Col_GeoLocation__Longitude__s,Col__c,Col_Url__c,Col_Time__c,Col_Text__c)
+
+CustomObjectData={"Name":"aman","Col_Timestamp__c":"2023-06-14T07:04:56.000+0000","Col_Date__c":"2023-06-14",\
+  "Col_Currency__c":123.456,"Col_Email__c":"ankit2627@gmail.com","Col_Number__c":1008.0,\
```

Comment:
Don't provide personal email here. Just take an instance of a dummy email.

Reply:
done